pegasus/sites.v3/code.org/layouts/default.haml

```diff
@@ -66,8 +66,13 @@
 
 -if header['theme'] == 'responsive'
   .desktop-feature
+    -if header['footer'] == 'suppl_footer'
```

Comment:
seems like it could be a bit more descriptive/specific.  Alternatively, header could actually specify an array of supplemental views to be added if we want this to be more generic.